samples/bluetooth/peripheral/src/main.c

```diff
@@ -22,9 +22,9 @@
 #include <bluetooth/uuid.h>
 #include <bluetooth/gatt.h>
 
+#include <bluetooth/services/bas.h>
```

Comment:
This should also not be necessary, once the Kconfig option is set the service should be registered by stack itself.